rncryptor.cabal

```diff
@@ -31,8 +31,8 @@ library
     , random >= 1.0.0.1
     , QuickCheck >= 2.6 && < 2.9
     , io-streams >= 1.2.0.0
-    , cipher-aes >= 0.2.9
-    , pbkdf >= 1.1.1.1
+    , cryptonite
```

Comment:
Hey Tom,

It's probably because the stack resolver used is a tad too old. Try using
one of the  latest ones (example lts-6.5) and that should work ;)

On 19 Aug 2016 8:52 p.m., "tomtitchener" <notifications@github.com> wrote:

> In rncryptor.cabal
> <https://github.com/RNCryptor/rncryptor-hs/pull/12#discussion_r75534005>:
>
> > @@ -31,8 +31,8 @@ library
> >      , random >= 1.0.0.1
> >      , QuickCheck >= 2.6 && < 2.9
> >      , io-streams >= 1.2.0.0
> > -    , cipher-aes >= 0.2.9
> > -    , pbkdf >= 1.1.1.1
> > +    , cryptonite
>
> adding cryptonite >= 0.15 gave me a stack error:
>
> -- Failure when adding dependencies:
>
> cryptonite: needed (>=0.15), 0.6 found (latest applicable is 0.19)
> needed for package rncryptor-0.0.2.3
>
> Adding cryptonite >= 0.6 works. Is that ok?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/RNCryptor/rncryptor-hs/pull/12/files/e044e6f5c5735c3575513b5d0ad7bc438b2c42e9#r75534005>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAa-s3k60snAyitSjrQ_61Z77fDKdi2Gks5qhfuGgaJpZM4JkuCD>
> .
>